src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php

```diff
@@ -69,16 +71,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $schema = new RepositorySchema;
         try {
             if ($input->getOption('drop')) {
-                foreach ($schema->toDropSql($connection->getDatabasePlatform()) as $sql) {
-                    if (true === $input->getOption('dump-sql')) {
-                        $output->writeln($sql);
-                    } else {
-                        try {
-                            $connection->exec($sql);
-                        } catch (\Exception $e) {
-                            $output->writeln($e->getMessage());
+                try {
+                    foreach ($schema->toDropSql($connection->getDatabasePlatform()) as $sql) {
+                        if (true === $input->getOption('dump-sql')) {
+                            $output->writeln($sql);
+                        } else {
+                            try {
+                                $connection->exec($sql);
+                            } catch (\Exception $e) {
+                                $output->writeln($e->getMessage());
```

Comment:
ups, did we really want to merge this like this? why would we then even care about catching the DBALException below?